.github/actions/build-avr/build.sh

```diff
@@ -0,0 +1,19 @@
+set -e
```

Comment:
There is a precedent for scripts of this nature to be named `entrypoint.sh`.

This can be observed in the GitHub documentation.
https://docs.github.com/en/actions/creating-actions/dockerfile-support-for-github-actions#entrypoint

Reply:
that seems like a good idea for consistency. 👍